robocop/checkers/naming.py

```diff
@@ -662,3 +669,60 @@ def find_similar_variables(self, tokens, node):
                     col=token.col_offset,
                 )
             self.variables[normalized_token].add(token.value)
+
+
+class DeprecatedStatementChecker(VisitorChecker):
+    """Checker for finding deprecated statments."""
+
+    reports = (
+        "deprecated-statement",
+    )
+    # deprecated:alternative
+    deprecated_statements_rf3 = {}
+    deprecated_statements_rf4 = {"runkeywordunless": "IF/ELSE",
+                                 "runkeywordif":     "IF/ELSE"
+                                 }
+    deprecated_statements_rf5 = {"runkeywordunless": "IF/ELSE"}
+
+    def visit_SuiteSetup(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.name, node)
+
+    def visit_TestSetup(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.name, node)
+
+    def visit_Setup(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.name, node)
+
+    def visit_SuiteTeardown(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.name, node)
+
+    def visit_TestTeardown(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.name, node)
+
+    def visit_Teardown(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.name, node)
+
+    def visit_TestCase(self, node):  # noqa
+        self.generic_visit(node)
+
+    def visit_KeywordCall(self, node):  # noqa
+        self.check_if_keyword_is_deprecated(node.keyword, node)
+
+    def check_if_keyword_is_deprecated(self, keyword_name, node):
+        normalized_keyword_name = normalize_robot_name(keyword_name, remove_prefix="builtin.")
+        deprecated_statements = self.deprecated_statements_rf3
+        if ROBOT_VERSION.major == 4:
+            deprecated_statements = self.deprecated_statements_rf4
+        elif ROBOT_VERSION.major == 5:
+            deprecated_statements = self.deprecated_statements_rf5
+        if deprecated_statements.get(normalized_keyword_name, None):
+            alternative = f"'{deprecated_statements.get(normalized_keyword_name)}'"
```

Comment:
```suggestion
            alternative = deprecated_statements[normalized_keyword_name]
```
And put '' in message itself:
... use '{{ alternative }}' instead